devops/base/server/deploy.yaml

```diff
@@ -28,22 +28,22 @@ spec:
   selector:
     app: rocketchat
     role: server
-  type: ClusterIP
----
-kind: PersistentVolumeClaim
-apiVersion: v1
-metadata:
-  labels:
-    app: rocketchat
-  name: rocketchat-uploads
-spec:
-  storageClassName: netapp-file-standard
-  accessModes:
-    - ReadWriteMany
-  resources:
-    requests:
-      storage: 1Gi
-  volumename: rocketchat-uploads
+#  type: ClusterIP
```

Comment:
is this PVC from somewhere else now?

Reply:
That PVC is just commented out for now, because if put it under the control of Argo CD it will have to be removed and recreated.  Personally, I would prefer to avoid that for now.